src/transformers/models/swin/modeling_swin.py

```diff
@@ -362,7 +459,7 @@ def prune_heads(self, heads):
     def forward(self, hidden_states, attention_mask=None, head_mask=None, output_attentions=False):
         self_outputs = self.self(hidden_states, attention_mask, head_mask, output_attentions)
         attention_output = self.output(self_outputs[0], hidden_states)
-        outputs = (attention_output,) + self_outputs[1:]  # add attentions if we output them
+        outputs = (attention_output, self_outputs[1]) if output_attentions else (attention_output,)
```

Comment:
Not sure if this changes anything?

Reply:
not sure either, working with tuple unnamed outputs is terrible. I'd leave it as it is to avoid further debugging sessions.

Reply:
Fine for me. But note taht any model in the library has the `# add attentions if we output them` comment

Reply:
That in this case is basically replaced by ` if output_attentions`. Let me refactor it back  :)

Reply:
Changed it back as it was before, that was probably a left over from when I have to return the `spatial_hidden_sizes`